maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/XCTest/RunningApp.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  RunningApp.swift
+//  maestro-driver-iosUITests
+//
+//  Created by Amanjeet Singh on 17/07/23.
+//
+
```

Comment:
Nit: iOS engineers tend to remove these bogus headers on file creation

Reply:
Done ✅